docs/src/main/sphinx/release/release-434.md

```diff
@@ -0,0 +1,36 @@
+# Release 434 (22 Nov 2023)
+
+## General
+
+* {{breaking}} Rename the `query.max-writer-tasks-count` configuration property
+  and the related `max_writer_tasks_count` session property to
+  `query.max-writer-task-count` and `max_writer_task_count`. ({issue}`19793`)
+* Prevent table function failure when its invocation is a first query run on a
+  worker. ({issue}`19849`)
+* Prevent sudden increases in PagePartitioner memory consumption in some
+  scenarios. ({issue}`19762`)
+
+## BigQuery connector
+
+* Add support for reading `json` columns. ({issue}`19790`)
+* Improve performance when writing rows. ({issue}`18897`)
+
+## Delta Lake connector
+
+* Improve performance when reading large checkpoint files on partitioned tables.
+  ({issue}`19588`)
+
+## Hive connector
+
+* Add support for columns that changed from `timestamp` to `date` type.
+  ({issue}`19513`)
+* Add tracing to metastore access. ({issue}`19712`)
```

Comment:
This is an internal detail. I don't think we need to document it here.

Reply:
Removed